### DISCUSSION TITLE
'requested module does not provide an export named' - Breaking change on 3.8

### DISCUSSION THREAD

**Discussion by LLCampos** [nuxt/nuxt#25379] [Questions]
Hi,

After upgrading to 3.8, when running `nuxt dev` I start getting errors, client-side, with the format

`The requested module '/_nuxt/something/something/types.ts' does not provide an export named 'NewsItem'`

Things work fine if I run `nuxt build; node .output/server/index.mjs` and were working fine on 3.7.

Any tips on what to check/try to solve this? :)

Thanks,

**Comment by larsalbrecht** [comment]
Have you tried to import the NewsItem as type?
Like:
`import type { NewsItem } from 'path-to-types.ts'`?

Can you provide more information?

**Reply by TheAlexLichter** [reply]
Yes, that is most likely the "issue" because `verbatimModuleSyntax` is on now.

https://github.com/nuxt/nuxt/pull/23447
